src/redux/configureStore.js

```diff
@@ -1,10 +1,11 @@
-import { combineReducers, createStore } from 'redux';
+import { combineReducers, createStore, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
 
 import booksReducer from './books/books';
 import categoriesReducer from './categories/categories';
 
 const rootReducer = combineReducers({ books: booksReducer, categories: categoriesReducer });
 
-const store = createStore(rootReducer);
+const store = createStore(rootReducer, applyMiddleware(thunk));
```

Comment:
Well done 🚀 `redux-thunk` is added to the project as a package and used as a middleware 🔥